### PR TITLE
uploading scripts for scale migration using ASR

### PR DESCRIPTION
**Pull request by snehaamicrosoft**

### CODE REVIEW

**Comment by snehaamicrosoft** [comment]
#sign-off

**Comment by rayne-wiselman** [comment]
@sptramer @snehaamicrosoft   I work with Sneha on content for Migrate and Site Recovery. Per our conversation, Sneha has created an azure-migrate folder and scripts to go in it. thx

**Comment by snehaamicrosoft** [comment]
@sptramer Just to add to Rayne's comments, we have validated the scripts and they work smoothly. We are  planning to announce them at Ready (give a demo at the migration session and a link to the repo), would appreciate if you could merge it soon. Thanks a lot!

File: azure-migrate/VMware migration using ASR/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
+{
```
Comment (sptramer) [inline azure-migrate/VMware migration using ASR/.vscode/launch.json]: .vscode files should not be checked in.

File: azure-migrate/VMware migration using ASR/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (sptramer) [inline azure-migrate/VMware migration using ASR/LICENSE]: All content is already covered by the root `LICENSE` file (which is MIT). Remove.

**Comment by rayne-wiselman** [comment]
@snehaamicrosoft Hi Sneha, use vmware-migrate-with-site-recovery. ASR isn't an approved acronym.

File: azure-migrate/VMware migration using ASR/LICENSE (hunk 2 above)
Comment (snehaamicrosoft) [reply azure-migrate/VMware migration using ASR/LICENSE]: Done

**Comment by snehaamicrosoft** [comment]
@sptramer I have removed the vscode and License files as per the review comments. Also have renamed the folder as per the suggestion. Kindly take a look.

**Comment by sptramer** [comment]
LGTM - and thank you for including a sample CSV. It's uncommon for scripts to have dependent files like external data or ARM templates submitted along with them and that's going to become a new condition for acceptance.

**Comment by snehaamicrosoft** [comment]
Thanks @sptramer, appreciate your quick turnaround on this.

**Pull request merged**